modules/apps/change-tracking/change-tracking-service/src/main/java/com/liferay/change/tracking/internal/spi/history/CTCollectionHistoryProviderRegistryImpl.java

```diff
@@ -35,7 +36,14 @@ public CTCollectionHistoryProvider<?> getCTCollectionHistoryProvider(
 			return null;
 		}
 
-		return _serviceTrackerMap.getService(className.getValue());
+		CTCollectionHistoryProvider<?> ctCollectionHistoryProvider =
+			_serviceTrackerMap.getService(className.getValue());
+
+		if (ctCollectionHistoryProvider == null) {
```

Comment:
Can you change it so that DefaultCTCollectionHistoryProvider is only created once